ha/setup/create_pacemaker_resources.py

```diff
@@ -0,0 +1,251 @@
+#!/bin/python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import argparse
+from ha.execute import SimpleCommand
+from cortx.utils.log import Log
+
+
+def cib_push(cib_xml):
+    """Shortcut to avoid boilerplate pushing CIB file."""
+    cmd_push = f"pcs cluster cib-push {cib_xml} --config"
+    SimpleCommand().run_cmd(cmd_push)
+
+
+def cib_get(cib_xml):
+    """Generate CIB file using pcs."""
+    cmd= f"pcs cluster cib {cib_xml}"
+    SimpleCommand().run_cmd(cmd)
+
+    return cib_xml
+
+
+def motr_hax(cib_xml, push=False):
+    """Create resources that belong to motr group and clone the group."""
```

Comment:
There should be one group for motr and s3 i.e. io_group. Current grouping will work in one node but there will be problem in multi-node.

Reply:
I added s3 background consumer to io_group  - is this correct? 
Reason: it is co-located with s3server and also runs on every node.